### PR TITLE
Prune vagrant environment after test

### PR DESCRIPTION
**Pull request by cynepco3hahue**
I hope it will fix our CI environments

Signed-off-by: Lukianov Artyom <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
ci test please

File: automation/check-patch.sh [hunk 1]
```diff
@@ -95,6 +95,8 @@ cleanup() {
     # Otherwise the environment cleanup code will clean them up as well as the
     # Docker data disks that we want to keep around.
     vagrant destroy
+    vagrant global-status --prune
+    ps aux | grep ruby | awk '{print $2}' | xargs kill -9
```
Comment (davidvossel) [inline automation/check-patch.sh]: can you put in a comment that provides some background for why this 'kill' of ruby pids is necessary

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
I can see it start to be better from `vagrant` perspective, but I still sometimes receive 
```
 + /home/jenkins/workspace/kubevirt_kubevirt_standard-check-pr/dockerize -wait tcp://192.168.121.186:6443 -timeout 300s
11:39:06 [check-patch.el7.x86_64] 2018/02/21 11:39:06 Waiting for host: 192.168.121.186:6443
11:39:10 [check-patch.el7.x86_64] 2018/02/21 11:39:09 Problem with dial: dial tcp 192.168.121.186:6443: getsockopt: no route to host. Sleeping 5s
11:39:18 [check-patch.el7.x86_64] 2018/02/21 11:39:17 Problem with dial: dial tcp 192.168.121.186:6443: getsockopt: no route to host. Sleeping 5s
```

**Comment by cynepco3hahue** [comment]
I think Roman containerized qemu will solve issues, so I close this PR

**Pull request closed without merging**